scripts/west_commands/sign.py

```diff
@@ -111,6 +111,8 @@ def do_add_parser(self, parser_adder):
                            help='''path to the tool itself, if needed''')
         group.add_argument('-D', '--tool-data', default=None,
                            help='''path to tool data/configuration directory, if needed''')
+        group.add_argument('-c', '--use-code-partition', action='store_true', default=False,
+                           help='''use zephyr,code-partition chosen DT node insead of default''')
```

Comment:
Spelling mistake, `insead` --> `instead`.

But I think I prefer just to state what the argument does, without the need to tell `instead of`.
```suggestion
                           help='''Sign zephyr,code-partition chosen DT node.''')
```